tests/drivers/spi/spi_loopback/boards/nucleo_f103rb.conf

```diff
@@ -1,2 +1,3 @@
 CONFIG_SPI_LOOPBACK_DRV_NAME="SPI_1"
 CONFIG_SPI_STM32_INTERRUPT=y
+CONFIG_SPI_LOOPBACK_MODE_LOOP=n
```

Comment:
According to https://docs.zephyrproject.org/1.14.0/guides/kconfig/index.html#redundant-defaults, bool default to n.
So we should be able to remove `CONFIG_SPI_LOOPBACK_MODE_LOOP=n` from all boards.conf files

Reply:
removing `CONFIG_SPI_LOOPBACK_MODE_LOOP=n`
this could be completed with a 
```
    extra_configs:
      - CONFIG_SPI_STM32_INTERRUPT=y
```
in the testcase.yaml